README.md

```diff
@@ -17,93 +15,128 @@ Integration tests use [Cypress](https://www.cypress.io) and employ [Gherkin](htt
 
 ## To use Prepare a case, follow these steps:
 
-Install dependencies using 
+Build, install and run with the API mocked and JS/CSS rebuilding on-the-fly using the usual docker compose commands:
 
 ```
-npm i
+docker compose build
+DISPLAY= docker compose up --build
+DISPLAY= docker compose up
 ```
 
-And then, to build the assets and start the app with (substituting in a valid client secret for the dev environment)
+Note the DISPLAY= .This is required to prevent passing your X DISPLAY (on Mac and Linux) into Docker, which Cypress requires for CI.
+The intention is to wire Cypress for interactive UI mode in the future which will use the same env key. 
+
+These commands are aliased as
+
 ```
-env API_CLIENT_SECRET={client-secret} COURT_CASE_SERVICE_URL=http://court-case-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk PAC_ENV=dev ENABLE_CASE_COMMENTS=true ENABLE_CASE_PROGRESS=true NOMIS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth npm run start
+npm start
+npm run build
 ```
 
-Or to use alongside a local instance of *court-case-service* 
+With node being inside a docker container so to are the `node_modules` and if you make a change to package.json you will
+need to rebuild the docker container. Rebuilding the containers can be is fairly slow but you shouldn't need to do it day-to-day.
+
+Note that changes to wiremocks will not auto refresh, you'll need to restart the specific wiremock container.
```

Comment:
```
nodemon --watch mappings -e json --exec "curl -X POST http://localhost:9091/__admin/mappings/reset"
```
works for watching wiremocks and will reload into wiremock server

Reply:
It's a follow on piece of work. Feel free to do it!

I'm not sure if wiremock is node based, I think it's Java, in which case node won't be inside the official wiremock docker image.